x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/calculations/moving_average.tsx

```diff
@@ -174,7 +175,12 @@ const MovingAveragePopup = () => {
     <HelpPopover
       anchorPosition="upCenter"
       button={
-        <HelpPopoverButton onClick={() => setIsPopoverOpen(!isPopoverOpen)}>
+        <HelpPopoverButton
```

Comment:
Have you considered using the `TrackApplicationView` component?

Here are the docs of the API I'm talking about: https://github.com/elastic/kibana/blob/master/src/plugins/kibana_usage_collection/server/collectors/application_usage/README.md#tracking-sub-views-inside-your-application

Reply:
Thanks for letting me know, this looks super interesting. Due to the structure of Lens this might not be super straight forward to implement, but I will definitely make a note to use this once Lens is ready. We should probably do it together with https://github.com/elastic/kibana/issues/48352